doc/postprocess.py

```diff
@@ -39,26 +39,27 @@ def process_html(fn, lines):
 
 def process_tex(lines):
     """
-    Remove unnecessary section titles from the LaTeX file.
+    Remove unnecessary section titles from the LaTeX file. (not anymore!)
```

Comment:
Always remove code, don't comment out.

Reply:
commented out lines removed now

Starting with Sphinx 1.7.3, there will thus be no need for this post-processing of tex file.